src/app/shared/enum/provider.ts

```diff
@@ -52,4 +52,10 @@ export enum ProviderWorkshopSameValues {
   website = 'website',
   facebook = 'facebook',
   instagram = 'instagram'
+}
+
+export enum InstitutionTypes {
+  Complex = 'Комплексний',
+  Profile = 'Профільний',
+  Specialized = 'Спеціалізований'
```

Comment:
move it from this file (provider-workshops-same-values is not for storing such enums) to provider.ts where there are all values that needed for provider